packages/components/aside/src/aside.js

```diff
@@ -0,0 +1,67 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './aside.css';
+import '@tradeshift/elements.button';
+import '@tradeshift/elements.cover';
+
+customElementDefineHelper(
+	'ts-aside',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				title: { type: String, attribute: 'data-title' },
```

Comment:
You removed the `dir` property?

Reply:
Yes, After merge your stuff. I will put `dir` property into core